### PR TITLE
Add top layers

### PR DESCRIPTION
**Pull request by jsalbert**
- Added support for `Sigmoid` apart from `Softmax` Activation, as `sigmoid` in the options
- Corrected multi top label addition, you can pass a list of layers to insert on top of the model now
- Introduced an option (`dropout_rate`) for adding dropout before the ending `Dense` layer

### CODE REVIEW

File: keras_trainer/trainer.py [hunk 1]
```diff
@@ -41,6 +41,8 @@ class Trainer(object):
         'momentum': {'type': float, 'default': 0.9},
         'sgd_lr': {'type': float, 'default': 0.01},
         'pooling': {'type': str, 'default': 'avg'},
+        'sigmoid': {'type': bool, 'default': False},
```
Comment (stephensolis) [inline keras_trainer/trainer.py]: Maybe better to have `activation` which can be `'sigmoid'` or `'softmax'`

Comment (jsalbert) [reply keras_trainer/trainer.py]: True

File: keras_trainer/trainer.py [hunk 2]
```diff
@@ -146,10 +148,33 @@ def run(self):
...
+                # Set Activation Layer
+                if self.sigmoid:
+                    self.top_layers.append(Activation('sigmoid', name='act_sigmoid'))
+                else:
+                    self.top_layers.append(Activation('softmax', name='act_softmax'))
+
+            # Layer Assembly
+            for i, layer in enumerate(self.top_layers):
+                if i == 0:
+                    self.top_layers[i] == layer(self.model.output)
```
Comment (jsalbert) [inline keras_trainer/trainer.py]: omg I put double == 👮 !

**Comment by iamrosmarin** [comment]
Looks awesome @jsalbert 🎉
